### DISCUSSION TITLE
IPBan trying to connect to server on the internet 104.26.14.115:https

### DISCUSSION THREAD

**Discussion by mindracer13** [DigitalRuby/IPBan#307] [General]
Can someone explain to me the purpose of IBAN trying to connect to that webserver? I have banned outgoing internet access on my machine and IPBAN tries non stop to access that server/site

**Comment by jjxtra** [comment]
That's a cloudflare ip. Set `UseDefaultBannedIPAddressHandler` to false in ipban.config to disable anonymous ip address submissions to the ipban database.
